assets/src/dashboard/theme.js

```diff
@@ -30,17 +30,37 @@ export const GlobalStyle = createGlobalStyle`
 	*::before {
 		box-sizing: border-box;
     }
-    
-    h1 {
-      font-family: 'Google Sans', 'Roboto', sans-serif;
-    }
 `;
 
 export function useTheme() {
   return useContext(ThemeContext);
 }
 
 const theme = {
+  colors: {
```

Comment:
Do we expect a dark mode? @samitron7 , if so we should be more generic about the naming of colors

Reply:
From Sam 
> Dashboard was purposely left as white so that we can differentiate it from the editor. It'll continue to be white.

So we do not need to worry about dark themes.

Reply:
No darkmode for the dashboard, as confirmed in slack by sam
<img width="316" alt="Screen Shot 2020-03-16 at 2 27 23 PM" src="https://user-images.githubusercontent.com/10720454/76801127-45abe200-6792-11ea-91f6-9291eb312ad5.png">